package-res/ccc/core/base/axis/color-axis.js

```diff
@@ -47,10 +52,16 @@ def('pvc.visual.ColorAxis', pvc_Axis.extend({
             // and if so, transform the color scheme.
             // If the user specified the colors,
             // do not apply default color transforms...
+
+            // CDF603 
+            // If there is a preserved map
+            // do not apply default color transforms 
             var optSpecified = this.option.isSpecified,
                 applyTransf = (this.scaleType !== 'discrete') ||
-                    optSpecified('Transform') ||
-                    (!optSpecified('Colors') && !optSpecified('Map'));
+                                optSpecified('Transform')     ||
+                              (!optSpecified('Colors')   && 
+                               !optSpecified('Map')      &&
+                               !this.Map);
```

Comment:
Please, don't create public properties or use uppercase names for anything but "class" names. Call it `_preservedMap`, for example.